Gordon360/Models/CCT/Housing/Preference.cs

```diff
@@ -0,0 +1,25 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[PrimaryKey("ApplicationID", "Preference1")]
+[Table("Preference", Schema = "Housing")]
+public partial class Preference
+{
+    [Key]
+    [StringLength(255)]
+    [Unicode(false)]
+    public string ApplicationID { get; set; }
```

Comment:
I would highly recommend implementing foreign keys in your relationship columns such as this one. You wouldn't want invalid mappings here.